commit d988da6eee8422774dff364050bf431b843a714a
Author: Arkadiusz Miskiewicz <arekm@maven.pl>
Date:   Thu Dec 13 00:09:08 2007 +0200

    Xprint: Clean up generated files

    Remember to clean generated wrapper files.
    (cherry picked from commit 977fcdea8198906936a64b8117e6a6d027c617e3)

diff --git a/hw/xprint/Makefile.am b/hw/xprint/Makefile.am
index dc8764a..f834966 100644
--- a/hw/xprint/Makefile.am
+++ b/hw/xprint/Makefile.am
@@ -41,3 +41,5 @@ Xprt_SOURCES =			\
         $(top_srcdir)/fb/fbcmap_mi.c
 
 EXTRA_DIST = ValTree.c
+
+CLEANFILES = miinitext-wrapper.c dpmsstubs-wrapper.c
